packages/chat-client/src/client.ts

```diff
@@ -409,6 +446,12 @@ export class ChatClient extends IChatClient {
         });
       }
 
+      const historyClient = new HistoryClient(this.core);
+      await historyClient.registerTags({
+        relayUrl: this.core.relayUrl || RELAYER_DEFAULT_RELAY_URL,
+        tags: ["2000", "2001", "2002", "2003", "2004", "2005", "2006", "2007"],
```

Comment:
Would be cleaner to create util function that gives you all the chatClient tags from https://github.com/WalletConnect/chat-client-js/blob/main/packages/chat-client/src/constants/engine.ts#L13 instead of hard-coding tags by hand here

Reply:
I wanted to write them by hand because I didn't want all of them - just realized I added `2006` & `2007` accidentally